gulp/config.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+var fs = require( 'fs' );
+
+/**
+ * Set up file paths
+ */
+var loc = {
+  src:  './src',
+  dist: './dist',
+  lib:  JSON.parse( fs.readFileSync( './.bowerrc' ) ).directory, // eslint-disable-line
```

Comment:
Would be better to be updated to include specific rule being ignored. Here and throughout.